src/news/2019-03-22-OCAD-OE-week.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Floe at OCAD University's Open Education Week Event
+filename: OCAD-OE-week
+date: '2019-03-22'
+---
+Members of the Floe team recently participated in [OCAD University's](https://www.ocadu.ca/)
+[Open EducationWeek](https://www.openeducationweek.org/) event,
+[Guerilla Education! A Very Hands-On Open Education Week Event](https://www2.ocadu.ca/event/guerrilla-education-a-very-hands-on-open-education-week-event)
+
+<img src="images/ocad-open-ed.png" alt="Photograph of a presentation slide on a screen with
+three blocks of text: 'Understanding: make it accessible and clear', 'Sharing: make it easy
+to adapt and share', 'Participation & Inclusion: make it inviting, safe and sustainable.'" /></br/>
+
+**One of panelist Jairus Khan's slides, about elements of openness.**
```

Comment:
This is meant to be italicized, not bold text. Please also remove the line above it, since it's not meant to be a new paragraph